_posts/partners-hidora

```diff
@@ -0,0 +1,96 @@
+# Partner Highlight: Enjoy OpenSearch one click installation with Hidora
```

Comment:
Can you add the front-matter?   ([You can see how other posts are structured](https://github.com/opensearch-project/project-website/blob/main/_posts/2022-01-21-dashboards-plugins-intro.markdown)).

Example:
```
---
layout: post
title: "Partner Highlight: Enjoy OpenSearch one click installation with Hidora"
authors:
  - piellick 
date: 2022-01-21
categories:
  - partners
twittercard:
  description: "Hidora is an expert in providing custom solutions for all sizes of businesses. Hidora Cloud Hosting Solutions are just what you need if you want an efficient, secure and scalable cloud hosting solution that is tailored to your needs. We offer different plans that are scalable according to your specific requirements so you can find one that suits you best. Our PaaS help developers to host their applications in few minutes with DevOps philosophy."
---
```

Reply:
Also, you need an author bio in another file ([example](https://github.com/opensearch-project/project-website/blob/main/_authors/andhopp.markdown))